handlebars.runtime-v4.0.10.js

```diff
@@ -25,32 +25,33 @@ THE SOFTWARE.
 
 */
 (function webpackUniversalModuleDefinition(root, factory) {
```

Comment:
The handlebars runtime file should be excluded from prettier. Just revert all of the changes in this file

Reply:
Corrected. Does it show on your ned now?